schemas/transfer.js

```diff
@@ -0,0 +1,24 @@
+const transferTokenSchema = {
+  $id: '#/definitions/transferToken',
```

Comment:
nit: I think its a good idea to name the schemas for functions alike with the function names, would do "transferTokens" instead of "transferToken" here instead.

Reply:
[4909896](https://github.com/ar-io/testnet-contract/pull/87/commits/4909896b379038f9d1fb56e31e7cebe91b574123)

Reply:
sorry, i hit submit to soon, edited my comment 😅

Reply:
sure - i'll align in a separate PR

Reply:
the contract matches the spec based on the handler - the naming discrepancy is our function names. we can address broadly separately.